solutions/Day_2/Day_02.ipynb

```diff
@@ -0,0 +1,521 @@
+    "\n",
+    "\n",
+    "34,67,55,33,12,98\n",
+    "\n",
+    "\n",
+    "> **_Then, the output should be:_**\n",
+    "\n",
+    "\n",
+    "['34', '67', '55', '33', '12', '98']\n",
+    "('34', '67', '55', '33', '12', '98')\n",
+    "\n",
+    "\n",
+    "### Hints:\n",
+    "\n",
+    "> **_In case of input data being supplied to the question, it should be assumed to be a console input.tuple() method can convert list to tuple_**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "\n",
+    "\n",
+    "**Solutions:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "cell_id": "4e32dcd59b684071b99bd2c5f3aa35a1",
+    "deepnote_cell_type": "code",
+    "deepnote_to_be_reexecuted": false,
+    "execution_millis": 3434,
+    "execution_start": 1686044043983,
+    "source_hash": "a7d984a"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter a list of values separated by commas: 34,67,55,33,12,98\n",
+      "['34', '67', '55', '33', '12', '98'] ('34', '67', '55', '33', '12', '98')\n"
+     ]
+    }
+   ],
+   "source": [
+    "lst = input('Enter a list of values separated by commas: ')\n",
+    "print(lst.split(','), tuple(lst.split(',')))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "cell_id": "bca140d35f9a41f08e4d0a3c564fb9a7",
+    "deepnote_cell_type": "markdown"
+   },
+   "source": [
+    "---\n",
+    "\n",
+    "# Question 5\n",
+    "\n",
+    "### **Question:**\n",
+    "\n",
+    "> **_Define a class which has at least two methods:_**\n",
+    ">\n",
+    "> - **_getString: to get a string from console input_**\n",
+    "> - **_printString: to print the string in upper case._**\n",
+    "\n",
+    "> **_Also please include simple test function to test the class methods._**\n",
+    "\n",
+    "### Hints:\n",
+    "\n",
+    "> **_Use **init** method to construct some parameters_**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "\n",
+    "\n",
+    "**Solutions:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 54,
+   "metadata": {
+    "cell_id": "57c958a9c20043b8b933375fa026ec8d",
+    "deepnote_cell_type": "code",
+    "deepnote_to_be_reexecuted": false,
+    "execution_millis": 3,
+    "execution_start": 1686044057888,
+    "source_hash": "b623e53d"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter a string: \n",
+      "NULL\n"
+     ]
+    }
+   ],
+   "source": [
+    "class StringOperations:\n",
+    "    def __init__(self):\n",
+    "      self.default_string = 'NULL'\n",
+    "\n",
+    "    def getString(self):\n",
+    "      user_str = input('Enter a string: ')\n",
+    "      return user_str\n",
+    "\n",
+    "    def printString(self, input_str):\n",
```

Comment:
`printString` method should not take argument. Just return value assigned in the `__init__` method, kile:
```python
def getString(self):
  self.default_string = input()

def printString(self):
  return self.default_string